contrib/drivers/drivers.go

```diff
@@ -102,7 +107,7 @@ func Init(resourceType string) VolumeDriver {
 		d = &spectrumscale.Driver{}
 		break
 	case config.HuaweiOceanStorBlockDriverType:
-		d = &oceanstor.Driver{}
+		d = &OceanStorDriver
```

Comment:
OceanStorDriver is defined as a global variable, so each time we call Init for Huawei OceanStor driver, we can get the same driver object, not create a new one as before.

Reply:
This's also for the concurrency circumstance, because each oceanstor driver maintains a connect to OceanStor storage, if there are big concurrency invokes, the connection number may exceed the limit of OceanStor.

Reply:
ok